bower.json

```diff
@@ -1,18 +1,6 @@
 {
   "name": "ember-faker",
   "dependencies": {
-    "handlebars": "~1.3.0",
-    "jquery": "^1.11.1",
-    "ember": "1.7.0",
-    "ember-data": "1.0.0-beta.10",
-    "ember-resolver": "~0.1.7",
-    "loader.js": "stefanpenner/loader.js#1.0.1",
-    "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
-    "ember-load-initializers": "stefanpenner/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0",
-    "Faker": "~2.1.0"
+    "Faker": "4.1.0"
```

Comment:
@alanlalonde I tried importing Faker as an NPM module but unfortunately it's only available as a CommonJS module. I couldn't really find an easy way of importing it, in the same way the Bower component is imported (the NPM module does not export a built file). So I'd like to leave it as it is for now, and we can look into importing it as a NPM module in a future PR. In any case, this PR does quite some much needed modernizing!